distribution/docs/src/main/resources/content/_transformers/searchui-xformer.adoc

```diff
@@ -1,79 +0,0 @@
-:title: SearchUI Transformer
-:type: transformer
-:subtype: queryResponse
-:status: published
-:link: _searchui_transformer
-:summary: Provides results in html format and a convenient, simple querying user interface.
-
-The Simple SearchUI is a `QueryResponseTransformer` that provides results in html format and a convenient, simple querying user interface.
-It is primarily used as a test tool and verification of configuration.
-The left pane of the Simple SearchUI contains basic fields to query the Catalog and other Sources.
-The right pane consists of the results returned from the query.
-
-===== Installing Simple SearchUI
-
-This transformer is installed by default with a standard installation in the ${ddf-catalog} application as the feature, `catalog-transformer-ui`.
```

Comment:
Do we no longer include this type of information?

Reply:
It's no longer a transformer, so it felt right to get rid of it. 
Also the feature it describes here literally does not exist.

Reply:
Oh didn't realize this was for a transformer not the simple search itself. Cool 👍